modules/ROOT/pages/batch-reference.adoc

```diff
@@ -0,0 +1,193 @@
+= Batch Component Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Mule batch components are designed to manage bulk processing of records with minimal adjustments to default component settings. However, batch components are highly configurable to meet specific processing and performance requirements.
+
+A Batch Job component must contain at least one Batch Step component. Optionally, each Batch Step component can contain a single Batch Aggregator component. The Mule flow in the following figure shows batch components.
+
+image::batch-flow-generic-example.png[alt=Batch Processing Components in a Mule Flow within Studio]
+
+The Batch Job component splits the payload of the Mule message into records. Within the Batch Step and Batch Aggregator components, these records are available with the keyword `payload`, and Mule variables are also accessible through the `vars` keyword. However, Mule attributes are not accessible from within these components. Both components return `null` on `attributes`.
```

Comment:
there is some capitalization going on that I'm just going to trust you know should be used 🤣